### PR TITLE
added buyer and txHash

### PR DESCRIPTION
**Pull request by ed3ath**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)? https://github.com/CryptoBlades/cryptoblades/issues/664

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

(insert description here)

### CODE REVIEW

File: frontend/src/views/Market.vue [hunk 1]
```diff
@@ -2288,6 +2325,10 @@ export default Vue.extend({
       return input[0].toUpperCase() + input.slice(1);
     },
 
+    formatUrl(input: string): string {
+      return `<a href="${getConfigValue('blockExplorerUrls')[0]}/tx/${input}">${input.substr(0,4)}...${input.substr(-6)})}</a>`;
```
Comment (Chriserus) [inline frontend/src/views/Market.vue:2329]: Why are you returning template here instead of using template at the top and setting :href value dynamically?

**Review by Chriserus — APPROVED** [review]

File: frontend/src/views/Market.vue (hunk 1 above)
Comment (ed3ath) [reply frontend/src/views/Market.vue:2329]: sorry, im not that good with FE

Comment (Chriserus) [reply frontend/src/views/Market.vue:2329]: You can place <a> tag in the upper part (template) and set :href="" to the value you want

Comment (ed3ath) [reply frontend/src/views/Market.vue:2329]: to be honest, idk how xD

**Comment by Sellie96** [comment]
Closing PR as the market on this repo should be removed

**Pull request closed without merging**